### DISCUSSION TITLE
Choosing collaborators to train but also validate

### DISCUSSION THREAD

**Discussion by Linardos** [FETS-AI/Challenge#110] [Q&A] [answered]
It seems that when I use Custom Collaborator Training Selection the returned "training_collaborators" are the only ones to receive training as expected. However, it seems that all centers receive the task of validation, which means models are again downloaded and uploaded which is costly and might defeat the purpose of speeding up the overall speed of learning by omitting clients at certain rounds. Here an example where I exclude collaborator 4 but the task of validation is being received anyway:
![image](https://user-images.githubusercontent.com/26694607/126189562-bc2acec0-66fc-481b-a112-37a880960a10.png)

I only realized that recently. Is there a way to omit this validation as well? This would save download and upload speed.

**Comment by sarthakpati** [comment, marked as answer]
Hi Linardos,

Currently, there is no easy way to disable this, but we will keep it in mind for future iterations! Thank you for the feedback.
